ivy/functional/backends/tensorflow/experimental/statistical.py

```diff
@@ -171,20 +291,13 @@ def nanmedian(
 ) -> Union[tf.Tensor, tf.Variable]:
     if overwrite_input:
         copied_input = tf.identity(input)
-        return tfp.stats.percentile(
-            copied_input,
-            50.0,
-            axis=axis,
-            interpolation="midpoint",
-            keepdims=keepdims,
-        )
-    return tfp.stats.percentile(
-        input,
-        50.0,
-        axis=axis,
-        interpolation="midpoint",
-        keepdims=keepdims,
-    )
+        axis = tuple(axis) if isinstance(axis, list) else axis
```

Comment:
small point but why convert a list to a tuple only to later on convert it back to a list (inside `_nanmedian_helper`) 😅